gui/city_info.cc

```diff
@@ -197,6 +200,12 @@ void city_info_t::init()
 		allow_growth.pressed = city->get_citygrowth();
 		allow_growth.add_listener( this );
 		add_component(&allow_growth);
+
+		// add "change highlight button"
+		highlight.init( button_t::box_state_automatic | button_t::flexible, "Highlight");
```

Comment:
It is better to describe the purpose of this button.
```suggestion
		highlight.init( button_t::box_state_automatic | button_t::flexible, "Make building belong to");
```

Reply:
The button itself only highlights the city buildings, at the same time it is selected the swap tool is selected which does swap/"make city building belong to" (altho the tool itself is only selected when the player is the public player).
See `city_info_t::action_triggered`.
![image](https://github.com/user-attachments/assets/aaaefacc-c669-4c97-a21d-b6b4c45b4af2)

Reply:
In case of being a public player, highlighting the buildings is only the supplemental measure of selecting the city for the tool. The actual purpose is selecting the city for the tool, so the description should directly express it.
It may be helpful to alter the description text according to the selected player. When a not public player is selected, this button only highlights the buildings, so the description "Highlight" is appropriate for that case.